pkg/webhookconfig/status.go

```diff
@@ -97,13 +101,12 @@ func createStatusUpdateEvent(status string, eventGen event.Interface) {
 }
 
 func (vc statusControl) UpdateLastRequestTimestmap(new time.Time) error {
-	deploy, err := vc.deployClient.Get(context.TODO(), deployName, metav1.GetOptions{})
+
+	lease, err := vc.leaseClient.Get(context.TODO(), "kyverno", metav1.GetOptions{})
 	if err != nil {
-		vc.log.WithName("UpdateLastRequestTimestmap").Error(err, "failed to get deployment")
-		return err
+		log.Log.Info("Lease 'kyverno' not found. Starting clean-up...")
```

Comment:
Suggest using built-in logger with methos name for debugging `vc.log.WithName("UpdateLastRequestTimestmap").Error()`.